Samples/Accelerator via Podman on Linux/Tasks/Litium Accelerator/README.md

```diff
@@ -0,0 +1,255 @@
+Error: No such image: registry.litium.cloud/apps/direct-shipment:latest
+```
+
+Then you could try to pull the images manually via Podman. There's a risk that
+the Podman credentials doesn't transfer well to docker-compose, but once the
+images are downloaded, it should work just fine.
+
+```sh
+podman login registry.litium.cloud
+
+podman pull registry.litium.cloud/apps/direct-shipment:latest
+podman pull registry.litium.cloud/apps/direct-payment:latest
+```
+
+### docker-compose: `AccessDeniedException[/usr/share/elasticsearch/data/nodes]`
+
+If elasticsearch or mssql fails, you may have to do some `chmod`'ing:
+
+```sh
+sudo chmod -R 777 data volumes
+```
+
+(Yes, very surgical.)
+Then try `docker-compose up --abort-on-container-exit` again.
+
+## Configure Litium database
+
+This requires that the `docker-compose` is currently up and running.
+
+Open Azure Data Studio and connect to:
+
+- Connection type: `Microsoft SQL Server`
+- Server: `localhost,5434`
+- Authentication type: `SQL Login`
+- User name: `sa`
+- Password: `Pass@word`
+
+Then start a new SQL script and run the following:
+
+```sql
+CREATE DATABASE LitiumEducation;
+```
+
+The run the following Makefile targets:
+
+```sh
+make litium-db-update
+
+make litium-db-user
+```
+
+Then set the connection string by editing the
+`Src/Litium.Accelerator.Mvc/appsettings.json` file:
+
+```json
+{
+  "Litium": {
+    "Data": {
+      "ConnectionString": "Pooling=true;User Id=sa;Password=Pass@word;Database=LitiumEducation;Server=localhost,5434"
+```
+
+## Run the accelerator
+
+```sh
+make dotnet-run
+```
+
+Then navigate to: <https://localhost:10011>
+
+(port 10011 is the `direct-payment` service, that will then redirect you to
+<https://bookstore.localtest.me:5001/Litium/UI/login>)
+
+> Even though we've installed the certificates, there's a high chance your web
+> browser has it's own cert store which it doesn't trust. For now, just click
+> "Yes, continue anyway" in your browser.
+
+Log in with:
+
+- Username: `admin`
+- Password: `nimda`
+
+## Deploy "Bookstore"
+
+In the side panel, navigate to "Deployment" > "Accelerator", or visit this
+link:
+
+<https://bookstore.localtest.me:5001/Litium/UI/settings/extensions/AcceleratorDeployment/deployment>
+
+In the "Install Accelerator package" form, enter:
+
+- Name: `Bookstore`
+- Domain name: `bookstore.localtest.me`
+
+Click <kbd>Import</kbd>, and wait a couple of minutes. It's a slow process,
+and no logs are spat out in the process. But be patient.
+
+> During the import, you can visit <https://bookstore.localtest.me:5001/Litium/UI/media>
+> and see the `0 Files 0 Images 0 Movies 0 Others` go up as it gets closer to
+> finishing the import.
+>
+> By the end, it should say something like `366 Files 365 Images 1 Movies 0 Others`
+
+Now you can visit <https://bookstore.localtest.me:5001> and see the template
+site!
+
+:warning: Please note that no products are visible yet, as that relies on the
```

Comment:
Did not know that you could add icon like this warning, great tip, thanks :-)

Reply:
It's just an emoji shortcode :)
For example, here's a full list:
https://gist.github.com/rxaviers/7360908